manage.py

```diff
@@ -3,8 +3,27 @@
 import os
 import sys
 
+import django
+from django.conf import settings
+from django.contrib.auth import get_user_model
 
-def main():
+
+def create_super_user() -> None:
+    """Create default super user for development mode."""
+    django.setup()
+    username = os.environ.get("SUPERUSER_USERNAME", "admin")
+    password = os.environ.get("SUPERUSER_PASSWORD", "admin")
+
+    user = get_user_model()
+
+    if not user.objects.filter(username=username).exists():
```

Comment:
We also want to check if any superuser exists already (created manually, for example) and not create the default one if it does

Reply:
the superuser is being created only during development and it should not cause any problems as the user can specify superuser username and password as environment variables.

Reply:
Also, do we want to create superuser automatically in production also?

Reply:
Nope